test-configs.yaml

```
@@ -183,27 +183,47 @@ test_plans:
     params:
       job_timeout: '120'
 
-  ltp-crypto:
+  ltp-opts: &ltp-rootfs
     rootfs: debian_buster-ltp_nfs
     pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
     filters:
       - blocklist: *kselftest_defconfig_filter
-    params:
+    params: &ltp-params
       skip_install: "true"
-      tst_cmdfiles: "crypto"
+      tst_cmdfiles: ""
       job_timeout: '15'
       skipfile: skipfile-lkft.yaml
 
+  ltp-crypto:
+    <<: *ltp-rootfs
```

```suggestion
    <<: *ltp
```
following the suggestion above.

---

This PR could have one patch with the 2 suggestions above, with the `&ltp` anchor.

Then one patch for `ltp-mm` which appears to be working fine.